tests/TestFiles/Extensions/GADTs1.hs

```diff
@@ -36,51 +27,111 @@ lengthList2 (Conss _ xs) = 1 + lengthList2 xs
 --   error, called at src/G2/Initialization/KnownValues.hs:127:10 in g2-0.2.0.0-inplace:G2.Initialization.KnownValues
 
 
-data MyList a where
-  Ni :: MyList a
-  Cons :: a -> MyList a -> MyList a
+-- data ShapeArea a where
```

Comment:
Is there a reason to comment all this out rather than just delete it (I'd lean toward the latter so that the file can focus on just actual tests)?

Reply:
I see and I think it's ok to delete all the commented code in GADTs1.hs?